tracim/tracim/controllers/content.py

```diff
@@ -761,7 +772,8 @@ def post(self, label, parent_id=None, can_contain_folders=False, can_contain_thr
             logger.error(self, 'An unexpected exception has been catched. Look at the traceback below.')
             traceback.print_exc()
 
-            tg.flash(_('Folder not created: {}').format(e.with_traceback()), CST.STATUS_ERROR)
+            tb = sys.exc_info()[2]
+            tg.flash(_('Folder not created: {}').format(e.with_traceback(tb)), CST.STATUS_ERROR)
```

Comment:
Non relateed with content view deletion; But this line was errored by with_traceback missing parameter.